src/websocket-service.ts

```diff
@@ -1,32 +1,32 @@
-import ws from 'ws'
-import http from 'http'
-import setupWSConnection from './socket-message-handling';
+import ws from "ws";
+import http from "http";
+import setupSignalingConnection from "./utils/signaling-connection";
+import setupProviderConnection from "./utils/provider-connection";
 
 /**
  *
  * [`y-websocket`](https://github.com/yjs/y-websocket) provides a template
  * websocket services to interface with their websocket provider
  * https://github.com/yjs/y-websocket/blob/master/bin/server.js
- * 
+ *
  * [`y-webrtc`](https://github.com/yjs/y-webrtc) provides
  * a thin signalling backing service that connects to their
- * webrtc client provider https://github.com/yjs/y-webrtc/blob/master/bin/server.js 
+ * webrtc client provider https://github.com/yjs/y-webrtc/blob/master/bin/server.js
+ *
+ * TODO: Build this.
  *
- * TODO: Build this. 
- * 
  * Functions from both services should be combined here, to produce
  * a single websocket connection that provides both websocket
  * push changes, and initial document loads, as well as connecting
  * peers to each other.
- * 
+ *
  * Data persistance.
  *   The y-websocket interface implements a leveldb data layer interface.
  *   * What is leveldb?
  *   * Implement connector to some docker database.
  */
- 
 
-const host = process.env.HOST || '0.0.0.0'
+const host = process.env.HOST || "0.0.0.0";
```

Comment:
avoid touching lines you don't need to touch. If you need to make formatting or whitespace changes, keep those in their own commit.

Reply:
Not really sure how that happened to be honest, I'll revert that change back.